### ISSUE TITLE
agree on one coding convention

### ISSUE THREAD

**Issue by szborows** [szborows/yebash#3]
it is low-prio, but we definitely should agree on one coding convention ;)
my proposal:
- if/else/for blocks - always surrounded by curly braces
- space between lang keyword (e.g. if, else) and parenthesis
- no unneeded blank lines
- 4 spaces for indentation

more to come after you answer

**Comment by szborows** [comment]
of course we can use something off-the-shelf as well

**Comment by Mrokkk** [comment]
Yup. That's quite a good proposal. I think I can agree on these conventions :wink:
